common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -0,0 +1,146 @@
+.. _common-imu-notch-filtering:
+
+============================================================================
+Managing Gyro Noise with the Static Notch and Dynamic Harmonic Notch Filters
+============================================================================
+
+As :ref:`discussed<common-vibration-damping>`, managing vibration in ardupilot flight controller installations is extremely important in order to yield predictable control of an aircraft. Typically installations utilise mechanical vibration damping in order to remove the worst of the vibration. However, mechanical damping can only go so far and software filtering must be used to remove further noise. To the flight controller, vibration noise looks like any other disturbance (e.g. wind) that the flight controller must compensate for in order to control the aircraft. Ardupilot uses a :ref:`low-pass <copter:INS_GYRO_FILTER>` software filter to remove much of this remaining vibration noise, however, filtering has an unwanted side-effect - it removes *everything* including attitudinal and control information about the aircraft. Thus, while aggressive filtering can eliminate noise it can also reduce the control and responsiveness of the aircraft. This problem gets particularly acute on helicopters with very high levels of vibration noise at harmonics of the rotor speed and small, high powered copters with low-levels of natural mechanical damping that require greater control.
+
+For multicopters, virtually all vibrations originate from the motors and, importantly, quite a lot is known about this noise source as the majority of it is linked to the motor rotational frequency.  For helicopters, the vibrations are linked to the rotor speed.  It is thus theoretically possible to construct a software filter that targets *just* this noise, leaving all of the useful gyro information alone. This is what a *notch filter* does - it targets a narrow band of frequencies.
+
+Ardupilot has support for two different notch filters - a :ref:`static notch filter<common-imu-notch-filtering-static-notch>` that can be set at a fixed frequency and a dynamic notch filter that can be targeted at a range linked to the motor rotational frequency for multicopters or the rotor speed for helicopters.
+
+With the introduction of dynamic notch filtering, the need for static notch filtering is reduced, although it can be useful in targeting particular resonant frequencies (e.g. of the frame). We will therefore look at dynamic filtering first.
+
+-----------------
+Multicopter Setup
+-----------------
+
+Pre-Flight Setup
+=================
+
+Arducopter allows you to select the mechanism used for controlling the harmonic notch frequency. By default this mechanism is :ref:`throttle-based<common-imu-notch-filtering-throttle-based-setup>` since that will work on all copters. However, for copters with an rpm sensor or BLHeli ESC telemetry support, setup can be radically simpler:
+
```

Comment:
I think you added this link:  _common-dshot-blheli-telemetry: so it could be a hyperlink on the "BLHeli ESC....." wording...otherwise its not used anywhere that I have found...

Reply:
Fixed